### DISCUSSION TITLE
Partition Export Only

### DISCUSSION THREAD

**Discussion by jkthedba** [mydumper/mydumper#744] [Ideas]
If we have very big table multi terrabyte and we want to just export the most recent partitions to new RDS to reduce storage space of RDS, can we develop something for take partition exports using mydumper and import via myloader @david-ducos-percona

**Comment by jkthedba** [comment]
i dont see any option for exporting individual partitions in the manual

**Reply by davidducos** [reply]
Hi @jkthedba you have --where option and I'm working on a where option per table. We could add a partition option per table too in the future, it won't be hard to implement

**Reply by jkthedba** [reply]
thanks so much  @davidducos

**Comment by davidducos** [comment]
References:
https://github.com/mydumper/mydumper/issues/887
https://github.com/mydumper/mydumper/issues/1093
